src/pyscal/gasoil.py

```diff
@@ -80,8 +80,9 @@ def __init__(
 
         if swirr < 0:
             logger.warning(
-                f"Negative swirr value, {swirr}, detected. Negative values are allowed,"
-                " but you should ensure that this is intentional."
+                "Negative swirr value, %f, detected. Negative values are allowed,"
```

Comment:
I disagree with pylint on this. Using f-strings improves readability so much that it warrants skipping the lazy logging altogheter (and there is no real speed gain for simple string insertions like this).

Reply:
I totally agree with you and prefer f-string over % operator. The reason I changed it to % operator is to follow pattern in other parts of the code.

I have reverted the changes now and probably will do the same for other % operator.